parent/pom.xml

```diff
@@ -0,0 +1,516 @@
+                                    <pluginExecutionFilter>
+                                        <groupId>org.codehaus.gmaven</groupId>
+                                        <artifactId>gmaven-plugin</artifactId>
+                                        <versionRange>[1.0,)</versionRange>
+                                        <goals>
+                                            <goal>compile</goal>
+                                            <goal>execute</goal>
+                                            <goal>testCompile</goal>
+                                        </goals>
+                                    </pluginExecutionFilter>
+                                    <action>
+                                        <execute />
+                                    </action>
+                                </pluginExecution>
+                            </pluginExecutions>
+                        </lifecycleMappingMetadata>
+                    </configuration>
+                </plugin>
+
+                <plugin>
+                    <artifactId>maven-compiler-plugin</artifactId>
+                    <version>3.3</version>
+                    <configuration>
+                        <source>${javac.source}</source>
+                        <target>${javac.target}</target>
+                        <compilerArgument>-g</compilerArgument>
+                        <compilerArgument>-Xlint:-serial</compilerArgument>
+                    </configuration>
+                </plugin>
+
+                <plugin>
+                    <artifactId>maven-jar-plugin</artifactId>
+                    <version>2.6</version>
+                    <configuration>
+                            <!-- see http://maven.apache.org/shared/maven-archiver/index.html -->
+                        <archive>
+                            <index>true</index>
+                            <addMavenDescriptor>true</addMavenDescriptor>
+                            <manifest>
+                                <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
+                                <addDefaultSpecificationEntries>true</addDefaultSpecificationEntries>
+                            </manifest>
+                                <!-- Extra attributes added to the MANIFEST.MF file -->
+                            <manifestEntries>
+                                <Java-Version>${java.version}</Java-Version>
+                                <Java-VM-Version>${java.vm.version}</Java-VM-Version>
+                                <User-Name>${user.name}</User-Name>
+                                <!-- see Package constructor using Manifest -->
+                                <Release-Timestamp>${project.buildtimestamp}</Release-Timestamp>
+                                <Maven-Version>${maven.version}</Maven-Version>
+                            </manifestEntries>
+                        </archive>
+                    </configuration>
```

Comment:
Is any of this used yet? Maybe this parent pom has a bunch of stuff required by some other branch? It appears there's a lot of config that isn't needed for the "native" module.